packages/scoutgame/src/points/__tests__/refreshBuilderLevels.spec.ts

```diff
@@ -1,39 +1,95 @@
 import { prisma } from '@charmverse/core/prisma-client';
-import { getCurrentSeasonStart, getPreviousSeason } from '@packages/dates/utils';
-import { seedBuilders, writeSeededBuildersToDatabase } from '@packages/testing/deterministicBuildersData';
+import { jest } from '@jest/globals';
+import { getCurrentSeasonStart } from '@packages/dates/utils';
+import { mockBuilder, mockBuilderNft, mockNFTPurchaseEvent } from '@packages/testing/database';
+import {
+  seedBuildersGemPayouts,
+  writeSeededBuildersGemPayoutsToDatabase
+} from '@packages/testing/deterministicBuildersGemPayoutsData';
+import { prettyPrint } from '@packages/utils/strings';
 
 import { refreshBuilderLevels } from '../refreshBuilderLevels';
 
+jest.useFakeTimers();
+
 describe.skip('refreshBuilderLevels', () => {
-  it('should refresh builder levels in the database', async () => {
-    const { builders, weeks } = seedBuilders({ season: getPreviousSeason(getCurrentSeasonStart()) });
+  beforeEach(() => {
+    jest.setSystemTime(new Date('2025-01-25'));
```

Comment:
I don't want us to start using this. pattern. please change the methods to pass in the dates properly. for example u can pass in "now"